content/content-tool/tool/src/webapp/vm/content/sakai_resources_list.vm

```diff
@@ -37,7 +37,7 @@
             #else
 			<li class="firstToolBarItem"><span class="current">$clang.getString("list.site")</span></li>
             #end
-			#if($canDeleteResource)
+			#if($canDeleteResource && $showTrash)

```

Comment:
This was one of the testing instructions of the ticket:

"2/ set site.soft.deletion=false - check that you DO NOT see the Trash tab at the top of Resources? "

Thought the trash tab was directly related with the soft deletion feature? Otherwise, please update the jira ticket. I'll talk with Adam about that.